File: Python/2. Condicionales/Ejercicio #2.py
# 2. Hacer un programa que pida 3 numeros y determine cual de los 3 es mayor.

NUM1 = float(input("Valor #1: "))
NUM2 = float(input("Valor #2: "))
NUM3 = float(input("Valor #3: "))

if NUM1>NUM2 and NUM1>NUM3:
    print("Valor #1 es mayor")
elif NUM2>NUM1 and NUM2>NUM3:
    print("Valor #2 es mayor")
elif NUM3>NUM1 and NUM3>NUM2:
    print("Valor #3 es mayor")
else:
    print("Ninguno es mayor")

File: 1. C++/3. Estructuras Condicionales/Ejercicio #2.cpp
/*2. Escriba un programa que lea 3 numeros y determine cual de ellos es el mayor*/

#include<iostream>

using namespace std;

int main()
{
    int Num1, Num2, Num3;

    cout<<"Ingrese un valor 1: ";cin>>Num1;
    cout<<"Ingrese un valor 2: ";cin>>Num2;
    cout<<"Ingrese un valor 3: ";cin>>Num3;

    if(Num1 > Num2 && Num1 > Num3)  // conector 'o'(||), conector 'y'(&&)
    {
        cout<<"El numero 1 es el mayor"<<endl;
    }
    else if(Num2 > Num1 && Num2 > Num3)  // conector 'o'(||), conector 'y'(&&)
    {
        cout<<"El numero 2 es el mayor"<<endl;
    }
    else if(Num3 > Num1 && Num3 > Num2)
    {
        cout<<"El numero 3 es el mayor"<<endl;
    }
    else if(Num1 == Num2 && Num1 == Num3)
    {
        cout<<"Los numeros son iguales"<<endl;
    }
    else
    {
        cout<<"Dos valores son mayores"<<endl;
    }
    return 0;
}